torch/_tensor_docs.py

```diff
@@ -2771,6 +2772,18 @@ def callable(a, b) -> number
     >>> z.size()
     torch.Size([2, 8])
 
+    >>> a=torch.randn((1,2,3,4))
```

Comment:
spaces around `=` and after `,` please. Also, could you remove the extra parentheses for consistency with example above? :)